### PR TITLE
Update release notes and version number for 2.0.0 release

### PR DESCRIPTION
**Pull request by JeroenDeDauw**

### CODE REVIEW

**Comment by JeroenDeDauw** [comment]
@malberts before we tag, could you double-check that the "render by default" config flag works properly via in-wiki test?

File: README.md [hunk 1]
```diff
@@ -61,8 +61,8 @@ Example:
 Both `#embed` and `#bitbucket` can be customized with these parameters:
 
 * `lang`: (Optional) One of the [supported languages]. Only necessary if the language is not detected from the file extension.
-* `line`: (Optional) Show line numbers.
-* `specificLines`: (Optional) Show only specific lines. Can be a single line number or a range separated with a hyphen (-). Multiple line numbers or ranges can be separated by commas.
+* `lineNumbers`: (Optional) Show line numbers.
+* `lines`: (Optional) Show only specific lines. Can be a single line number or a range separated with a hyphen (-). Multiple line numbers or ranges can be separated by commas.
```
Comment (myousuffazal) [inline README.md:65]: As per the code changes, this should be showLines

File: src/UseCases/Embed/EmbedRequestBuilder.php [hunk 2]
```diff
@@ -25,7 +25,7 @@ public static function argumentsToRequest( array $arguments, bool $showEditButto
 		return new EmbedRequest(
 			fileUrl: $arguments[0],
 			language: is_string( $language ) ? $language : null,
-			showLineNumbers: is_bool( $line ) ? $line : null,
+			showLineNumbers: is_bool( $lineNumbers ) ? $lineNumbers : null,
 			showSpecificLines: is_string( $specificLines ) ? $specificLines : null,
```
Comment (myousuffazal) [inline src/UseCases/Embed/EmbedRequestBuilder.php:29]: Shouldn't we change all variables, remove 'specific', and add 'show' (where applicable) that will make it consistent across the code?

Comment (JeroenDeDauw) [reply src/UseCases/Embed/EmbedRequestBuilder.php:29]: That would be a bit cleaner but is not a blocker for the release

**Pull request merged**